crates/sui-types/src/sui_system_state/sui_system_state_inner_v1.rs

```diff
@@ -600,7 +600,7 @@ impl Default for SuiSystemStateInnerV1 {
             pending_active_validators: TableVec::default(),
             pending_removals: vec![],
             staking_pool_mappings: Table::default(),
-            inactive_pools: Table::default(),
+            inactive_validators: Table::default(),
```

Comment:
Field names `inactive_pools_id` and `inactive_pools_size` in `SuiSystemStateSummary` should be changed too then.